test_runner/fixtures/neon_fixtures.py

```diff
@@ -658,6 +698,22 @@ def ignore_postgres_log(path: str, _names):
 
         return self.env
 
+    def _mix_versions(self):
+        assert self.version_combination is not None, "version combination must be set"
+        for component, paths in COMPONENT_BINARIES.items():
+            directory = (
+                self.neon_binpath
+                if getattr(self.version_combination, component) == "new"
+                else self.compatibility_neon_binpath
+            )
+            for filename in paths:
+                destination = self.mixdir / filename
+                destination.unlink(missing_ok=True)
```

Comment:
Hmm, in which case we could have this file existing?

Reply:
We couldn't, removed